README.md

```diff
@@ -2,33 +2,46 @@
 
 # Kuzzle compatibility
 
-Versions 3.x of this plugin are compatible with Kuzzle v1.0.0-RC.8 and upper.
+Versions 4.x of this plugin are compatible with Kuzzle v1.0.0-RC.10 and upper.
 
-For older versions of Kuzzle, install v1.x or v2.x versions of this plugin instead.
+For older versions of Kuzzle, install v1.x, v2.x or v3.x versions of this plugin instead.
 
 # Plugin Local Password Authentication
 
 This plugin provides a local authentication with username/password with [passportjs module](http://passportjs.org/docs/username-password).
 
 By default, this plugin is already installed in Kuzzle.
 
-# Manifest
+# Configuration
 
-This plugin doesn't need any right.
+The default configuration is:
 
-# Configuration
+```json
+{
+  "secret": null,
+  "algorithm": "sha256",
+  "digest": "hex"
+}
+```
+
+All the configurations are used to set the behavior of the password hash.
+If `secret` is `null`, it will be generated automatically when the plugin is initalized for the first time.
+To change this configuration please refer to the [Plugin reference](http://docs.kuzzle.io/plugin-reference/#custom-plugin-configuration).
 
+Note: If this configuration is changed when users already exist, they won't be able to authenticate anymore.
```

Comment:
Well... not exactly since this configuration is applied only if there is no salt stored in the plugin storage.

Proposal:
`Note: the `secret` parameter is only used if there is no salt already configured in the database. If that's the case and if users are already stored, make sure that it matches the previously used salt`